src/js/view/utils/views-manager.js

```diff
@@ -78,19 +87,29 @@ if (isAndroidChrome && hasOrientation) {
 window.addEventListener('beforeunload', () => {
     document.removeEventListener('visibilitychange', onVisibilityChange);
     document.removeEventListener('webkitvisibilitychange', onVisibilityChange);
+    document.removeEventListener('webkitvisibilitychange', onVisibilityChange);
+    window.removeEventListener('scroll', onScroll);
 
     if (isAndroidChrome && hasOrientation) {
         window.screen.orientation.removeEventListener('change', onOrientationChange);
     }
 });
 
+window.addEventListener('scroll', onScroll);
+
 export default {
     add: function(view) {
         views.push(view);
     },
     remove: function(view) {
         removeFromGroup(view, views);
     },
+    addScrollHandler: function(handler) {
```

Comment:
want to unit test this but unsure of the best way to make `scrollHandlers` accessible to the outside world. is a synthetic scroll event possible with our setup? could test that way with mocked funcs

Reply:
There are some examples of synthetic browser events in ui-test.js:

https://github.com/jwplayer/jwplayer/blob/ac5c60ac26757d01378552ed99eba9934735a56d/test/unit/ui-test.js#L34